core/src/main/java/org/keycloak/util/TokenUtil.java

```diff
@@ -48,6 +48,11 @@
 
     public static final String TOKEN_TYPE_OFFLINE = "Offline";
 
+    public static final String TOKEN_TYPE_LOGOUT = "Logout";
+
+    public static final String TOKEN_BACKCHANNEL_LOGOUT_EVENT = "http://schemas.openid.net/event/backchannel-logout";
+
+    public static final String TOKEN_CLAIM_NONCE = "nonce";
```

Comment:
I would just reuse `org.keycloak.representations.IDToken#NONCE`.

Reply:
sure i will change that to the IDToken#NONCE

Reply:
done